rs/tests/driver/src/driver/farm.rs

```diff
@@ -455,8 +455,8 @@ impl GroupSpec {
         };
 
         // Acquire bazel's volatile status containing key value pairs like USER and CI_JOB_NAME:
-        let volatile_status_file_path = std::env::var("VOLATILE_STATUS_FILE_PATH")
-            .expect("Expected the environment variable VOLATILE_STATUS_FILE_PATH to be defined!");
+        let volatile_status_file_path = std::env::var("FARM_METADATA_PATH")
+            .expect("Expected the environment variable FARM_METADATA_PATH to be defined!");
         let volatile_status = read_dependency_to_string(&volatile_status_file_path)
```

Comment:
Consider renaming the variables `volatile_status_file_path` and `volatile_status` to `metadata_path` and `metadata` respectively. I would also remove the `FARM_` from the `FARM_METADATA_PATH` env var because we also want to use the same metadata with the k8s-based system-test backend.

Reply:
I've renamed the variables. Re. FARM_METADATA_PATH I think it makes sense, though I'd suggest waiting until we have the k8s tests using the variable (I'm hoping to have time to implement another solution by then)